docs/src/browsers.md

```diff
@@ -3,184 +3,80 @@ id: browsers
 title: "Browsers"
 ---
 
-Each version of Playwright needs specific versions of browser binaries to operate. Depending on the language you use, Playwright will either download these browsers at package install time for you, or you will need to use [Playwright CLI](./cli.md) to install these browsers.
+Each version of Playwright needs specific versions of browser binaries to operate. You will need to use Playwright CLI to install these browsers.
 
 With every release, Playwright updates the versions of the browsers it supports, so that the latest Playwright would support the latest browsers at any moment. It means that every time you update playwright, you might need to re-run the `install` CLI command.
 
-## Chromium
+## Update Playwright dependency
 
-For Google Chrome, Microsoft Edge and other Chromium-based browsers, by default, Playwright uses open source Chromium builds.
-Since Chromium project is ahead of the branded browsers, when the world is on Google Chrome N, Playwright already supports
-Chromium N+1 that will be released in Google Chrome and Microsoft Edge in a few weeks.
+By keeping your Playwright version up to date you will be able to test your app on the latest browser versions and catch failures before the latest browser version is released to the public.
 
-There is also a way to opt into using Google Chrome's or Microsoft Edge's branded builds for testing. For details
-on when to opt into stable channels, refer to the [Google Chrome & Microsoft Edge](#google-chrome--microsoft-edge) section below.
-
-## Firefox
-
-Playwright's Firefox version matches the recent [Firefox Stable](https://www.mozilla.org/en-US/firefox/new/)
-build.
-
-## WebKit
-
-Playwright's WebKit version matches the recent WebKit trunk build, before it is used in Apple Safari and
-other WebKit-based browsers. This gives a lot of lead time to react on the potential browser update issues.
-
-## Google Chrome & Microsoft Edge
-
-While Playwright can download and use the recent Chromium build, it can operate against the stock Google
-Chrome and Microsoft Edge browsers available on the machine (note that Playwright doesn't install them by
-default). In particular, current Playwright version will support Stable and Beta channels of these browsers.
-Here is how you can opt into using the stock browser:
-
-```js tab=js-js
-// @ts-check
-
-const { defineConfig } = require('@playwright/test');
-
-module.exports = defineConfig({
-  use: {
-    channel: 'chrome',
-  },
-});
+```bash js
+npm install -D @playwright/test@latest
```

Comment:
we need more code snippets here for the languages.

Reply:
can you send me these snippets as they are not in docs and I was told for Java its not as straight forward as just adding a command and in python it depends on how your project is setup etc so am not sure what to add here for other langs

Reply:
clarified offline, we only show this paragraph in Node.js